xml_serializable/lib/src/generator_factories/serializer_generator_factory.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:analyzer/dart/element/type.dart';
+import 'package:xml_serializable/src/serializer_generators/enum_serializer_generator.dart';
```

Comment:
Could we make this path relative to keep it consistent with the others in this file and the rest of the package?

```suggestion
import '../serializer_generators/enum_serializer_generator.dart';
```